sklearn/utils/estimator_checks.py

```diff
@@ -379,6 +385,28 @@ def check_estimator_sparse_data(name, Estimator):
             raise
 
 
+@ignore_warnings(category=(DeprecationWarning))
```

Comment:
You don't need the parentheses around ``DeprecationWarning``, they don't do anything.